### PR TITLE
doc: Add stickler-ci to the active tool list

### PR DESCRIPTION
**Pull request by markstory**
I maintain and operate stickler-ci and we offer hosted checkstyle integration with GitHub pull requests, with output being provided as check run annotations.

### CODE REVIEW

**Comment by pbludov** [comment]
@markstory the link is broken:
```
$ curl https://www.stickler-ci.com/
curl: (51) SSL: no alternative certificate subject name matches target host name 'www.stickler-ci.com'
```

**Comment by romani** [comment]
https://stickler-ci.com/

Works for me.

File: src/xdocs/index.xml.vm [hunk 1]
```diff
@@ -288,6 +288,21 @@
               </td>
               <td/>
             </tr>
+            <tr>
+              <td>
+                <a href="https://www.stickler-ci.com/">Stickler CI</a>
```
Comment (romani) [inline src/xdocs/index.xml.vm:293]: https://wheregoes.com/retracer.php

```
https://www.stickler-ci.com/
Error Tracing URL
Error: Timeout Error when fetching URL https://www.stickler-ci.com/
```

Comment (markstory) [reply src/xdocs/index.xml.vm:293]: :blush: Oops. Sorry about that.

**Review by pbludov — APPROVED** [review]

**Pull request merged**